build.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh -l
+set -e
+echo "::set-output name=javaversion::$(java -version)"
```

Comment:
What is the purpose of this line?

Reply:
See https://docs.github.com/en/actions/reference/workflow-commands-for-github-actions. It's a github actions syntax to capture variables that can be used in scripts..

Reply:
Okay. Thanks.